src/reviewers.rs

```diff
@@ -192,6 +193,7 @@ impl Reviewers {
         insert("pwalton", alias("pcwalton"));
         insert("quietmisdreqvus", alias("quietmisdreavus"));
         insert("rkruppe", alias("hanna-kruppe"));
+        insert("rustin170506", alias("0xpoe"));
```

Comment:
hi-rustin -(https://github.com/rust-lang/team/pull/1516)-> Rustin170506 -(https://github.com/rust-lang/team/pull/1818)-> 0xPoe

Ofc, various mailmaps still need to be updated (in the relevant repos).